awscli/customizations/configure/configure.py

```diff
@@ -39,7 +40,16 @@ class InteractivePrompter(object):
     def get_value(self, current_value, config_name, prompt_text=''):
         if config_name in ('aws_access_key_id', 'aws_secret_access_key'):
             current_value = mask_value(current_value)
-        response = raw_input("%s [%s]: " % (prompt_text, current_value))
```

Comment:
Would we ever not want this behavior?

Seems like we should fix this in compat rather than at the call site.